### PR TITLE
VSIX NGen + VS Insertion Setup

### PR DESCRIPTION
**Pull request by otawfik-ms**

### CODE REVIEW

**Comment by otawfik-ms** [comment]
@Microsoft/fsharp-compiler @KevinRansom

File: vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest [hunk 1]
```diff
@@ -5,10 +5,29 @@
...
     <DisplayName>Visual F# Tools</DisplayName>
     <Description xml:space="preserve">Deploy Visual F# Tools templates to Visual Studio</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\FSharp</ShortcutPath>
   </Metadata>
   <Installation>
     <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[15.0]" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="FsiAnyCPU.exe" />
```
Comment (KevinRansom) [inline vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest]: Does MicroBuild ensure that these are all signed before the vsix is made?

And is the resulting vsix signed?

Comment (KevinRansom) [inline vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest]: Does the VsixManifest not need to specify the optdata and sigdata files?

Comment (KevinRansom) [inline vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest]: should we also drop licence.txt

**Comment by KevinRansom** [comment]
Looks good to me.

File: vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest (hunk 1 above)
Comment (otawfik-ms) [inline vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest]: @KevinRansom 
Will ensure vsix binaries are signed before zipping.
License/optdata/sigdata is attached in the vsix as well (before that PR).

**Pull request merged**